@xen-orchestra/lite/src/libs/utils.ts

```diff
@@ -116,6 +116,19 @@ export function isHostRunning(host: XenApiHost) {
   }
 }
 
+export function getHostMemory(host: XenApiHost) {
+  try {
+    const metrics = useHostMetricsStore().getRecord(host.metrics);
+    const total = +metrics.memory_total;
+    return {
+      usage: total - +metrics.memory_free,
+      size: total,
+    };
+  } catch (_) {
+    return undefined;
```

Comment:
What kind of error could be thrown here? Shouldn't we at least make a `console.error` or something like this, to now when something bad happened?